### ISSUE TITLE
newly build packages disable routing for local network.

### ISSUE THREAD

**Issue by DocEmmetBrown** [tailscale/tailscale-qpkg#74]
Hey folks,
I tried to install the latest 1.36 package on my TS-451, and once properly configured, I lose inbound connectivity to my local IP, even though the Tailscale IP works
If I stop the tailscale daemon, my local IP pings again from my local network.
I can take a shell on the machine through Tailscale IP / alias, and from there I can ping my local network.
On the NAS, the routing table looks exactly the same with version 1.34.1 (which works as intended), and 1.36.0 (which doesn't).
I had the exact same issue experimenting with the 1.34.2 Pre-release package.
Happy to provide more details if needed.

**Comment by KilllerRabbbit** [comment]
I had the same problem with a few preleases I tried but with the official release [v1.36.0] (https://github.com/tailscale/tailscale-qpkg/releases/tag/v1.36.0) that didn't happen. 
There were some other problems with login that required ssh as the GUI was not really working in the beginning but no problems with network routing/isolation.

**Comment by talios** [comment]
I had this with this official release of `Tailscale_1.36.0-1_x86_64.qpkg` - GUI doesn't work (never did), tried running `tailscale up` again and reattaching - then I noticed it was only the local network that was dead.

Rolled back and all good again.

**Comment by DentonGentry** [comment]
> GUI doesn't work (never did)

What happens when you click on the Tailscale icon in the App Center? Does it open a new browser window? Gives an error in the browser? Something else?

> then I noticed it was only the local network that was dead.

Do you have any subnet routers on your tailnet? Might the QNAP be receiving routes from somewhere?

**Comment by talios** [comment]
When I open the GUI:

![image](https://user-images.githubusercontent.com/19093/215254563-707ac215-3210-4976-98a5-6da5ea452223.png)

Where would I find logs for that?

No subnet routers that I'm away of - just the QNAP, the Macbook, a raspberry pi and an iPhone.

I don't believe theres anything else that'd issue routes - ports, maybe, but not routes.

**Comment by DentonGentry** [comment]
The screenshot shows a browser connection to `nas50d992:9090/wailscaleweb`, but it is supposed to open a connection to `cgi-nas50d992:9090/cgi-bin/qpkg/Tailscale/index.cgi`

Is this using https://github.com/tailscale/tailscale-qpkg/releases/tag/v1.36.0 ?

**Comment by talios** [comment]
Ahh no it wasn't - that was using the earlier release as I'd rolled back due the local networking issue.  I'll try again.

Local networking/dns seems to be working again this time - I see the shortcut to Tailscale on the 'desktop' and the menu still point to the old URL, however manually changing the URL I get:

![image](https://user-images.githubusercontent.com/19093/215290722-950a97d6-2a3d-46f7-9d2f-aa95cecb8ef3.png)

```
Get "http://127.0.0.1:8080/cgi-bin/authLogin.cgi?sid=rdzanxjv": dial tcp 127.0.0.1:8080: connect: connection refused
```

**Comment by talios** [comment]
Interesting - without any changes I now seem to get:

```
http: named cookie not present
```

from that above link.

**Comment by davegoodfellow** [comment]
> Hey folks, I tried to install the latest 1.36 package on my TS-451, and once properly configured, I lose inbound connectivity to my local IP, even though the Tailscale IP works If I stop the tailscale daemon, my local IP pings again from my local network. I can take a shell on the machine through Tailscale IP / alias, and from there I can ping my local network. On the NAS, the routing table looks exactly the same with version 1.34.1 (which works as intended), and 1.36.0 (which doesn't). I had the exact same issue experimenting with the 1.34.2 Pre-release package. Happy to provide more details if needed.

I have the same problem.
QTS 5.0.0.2131 TVS-672N
Tailscale_1.36.0-1_x86_64.qpkg
GUI works fine when I start the app.
I then lose connection  to QNAP on 192.168.1.17 and can only connect on tailscale IP 100.xx.xx.xx
Do I need to add a static route?

**Comment by DocEmmetBrown** [comment]
@DentonGentry : just tried the new `Tailscale_1.36.1-1` on my TS-451, and had the exact same behaviour 🤔 . 
Immediately after enabling the device in the admin console, I loose all connectivity on my LAN IP. 
Reverting back to 1.34.1 and deleting / recreating the device in the console did make it work again, but I wonder if I have something special in my config 🤔 .
Happy to provide whatever details I can to help debug this issue.

**Comment by FlintyLemming** [comment]
@DocEmmetBrown 
When I keep the WebUI port is 5000 which is default, same as above
![CleanShot 2023-02-21 at 18 13 58@2x](https://user-images.githubusercontent.com/19165511/220316755-bfff3688-928f-4af2-9e99-ce7e56d978fc.png)
After change it into 8080, everything is fine.
![CleanShot 2023-02-21 at 18 15 49@2x](https://user-images.githubusercontent.com/19165511/220317116-2db8cb54-5793-4519-8b18-74c2a3b6ca22.png)
May it help you.

**Comment by DocEmmetBrown** [comment]
Hey @DentonGentry,
I did a bit of digging and I _think_ I understand better the problem: 

TL;DR : 
the newly built packages seem to have the `--accept-routes` flag enabled. 

On my network, I have one of my nodes that advertises my local network route (`192.168.166.0/24`). 
When I disable this route advertisement, my NAS works perfectly.
As soon as I enable the route advertisement, I loose all inbound connectivity to the LAN interface.

Here you have the (redacted) logs when enabling route sharing (through the admin UI) : 
```
2023/05/05 19:03:12 wgengine: Reconfig: configuring userspace WireGuard config (with 2/7 peers)
2023/05/05 19:03:12 wgengine: Reconfig: configuring router
2023/05/05 19:03:12 wgengine: Reconfig: configuring DNS
2023/05/05 19:03:12 dns: Set: {DefaultResolvers:[999.999.999.999] Routes:{beta.tailscale.net.:[] tailnet4242.ts.net.:[] ts.net.:[333.333.333.333 1234:123:1234::53]}+65arpa SearchDomains:[tailnet4242.ts.net. mygithubuser.github.beta.tailscale.net.] Hosts:16}
2023/05/05 19:03:12 dns: Resolvercfg: {Routes:{.:[999.999.999.999] ts.net.:[333.333.333.333 1234:123:1234::53]} Hosts:16 LocalDomains:[beta.tailscale.net. tailnet4242.ts.net.]+65arpa}
2023/05/05 19:03:12 dns: OScfg: {Nameservers:[100.100.100.100] SearchDomains:[tailnet4242.ts.net. mygithubuser.github.beta.tailscale.net.] MatchDomains:[] Hosts:[]}
2023/05/05 19:03:12 monitor: RTM_NEWROUTE: src=, dst=192.168.166.0/24, gw=, outif=21, table=52
```

and disabling route sharing
```
2023/05/05 19:03:23 wgengine: Reconfig: configuring userspace WireGuard config (with 1/7 peers)
2023/05/05 19:03:23 wgengine: Reconfig: configuring router
2023/05/05 19:03:23 monitor: RTM_DELROUTE: src=, dst=192.168.166.0/24, gw=, outif=21, table=52
2023/05/05 19:03:23 wgengine: Reconfig: configuring DNS
2023/05/05 19:03:23 dns: Set: {DefaultResolvers:[999.999.999.999] Routes:{beta.tailscale.net.:[] tailnet4242.ts.net.:[] ts.net.:[333.333.333.333 1234:123:1234::53]}+65arpa SearchDomains:[tailnet4242.ts.net. mygithubuser.github.beta.tailscale.net.] Hosts:16}
2023/05/05 19:03:23 dns: Resolvercfg: {Routes:{.:[999.999.999.999] ts.net.:[333.333.333.333 1234:123:1234::53]} Hosts:16 LocalDomains:[tailnet4242.ts.net. beta.tailscale.net.]+65arpa}
2023/05/05 19:03:23 dns: OScfg: {Nameservers:[100.100.100.100] SearchDomains:[tailnet4242.ts.net. mygithubuser.github.beta.tailscale.net.] MatchDomains:[] Hosts:[]}
```

Happy to share more details if that can help.

Cheers

**Comment by n1majne3** [comment]
seems still have issue in latest 1.48, lost LAN connect once enable tailscale, now use docker instead....

**Comment by wingcomm** [comment]
Same issue here still on 1.68.2.
